docs/okeps/okep-5094-layer2-transit-router.md

```diff
@@ -0,0 +1,981 @@
+to transit_router without issues. In general it will be similar to the layer3 routes and policies.
+
+These will be the routes and policies configured on the new transit_router
+```
+IPv4 Routes
+Route Table <main>:
+                10.96.0.1               203.203.0.2 dst-ip <-- api service
+               10.96.0.10               203.203.0.2 dst-ip <-- dns service
+               100.65.0.2                100.88.0.6 dst-ip <-- node1 traffic use node2 gr peer ip
+               100.65.0.3               100.88.0.10 dst-ip <-- node2 traffic use node2 gr peer ip
+               100.65.0.4               100.88.0.14 dst-ip <-- node3 traffic use node2 gr peer ip
+           203.203.0.0/16               100.88.0.14 src-ip <-- network egress traffic goes to local gateway router peer ip
+```
+```
+      1004 inport == "rtos-test12_namespace.scoped_ovn_layer2_switch" && ip4.dst == 172.18.0.3 /* test12_namespace.scoped_ovn_layer2_switch */         reroute               203.203.0.2
+      1004 inport == "rtos-test12_namespace.scoped_ovn_layer2_switch" && ip6.dst == fc00:f853:ccd:e793::3 /* test12_namespace.scoped_ovn_layer2_switch */         reroute           2010:100:200::2
+       102 (ip4.src == $a10466913729612642039 || ip4.src == $a13607449821398607916) && ip4.dst == $a3613486944346402462           allow
+       102 (ip4.src == $a10466913729612642039 || ip4.src == $a13607449821398607916) && ip4.dst == $a712973235162149816           allow               pkt_mark=1008
+       102 (ip6.src == $a2718358047735721557 || ip6.src == $a13607452020421864338) && ip6.dst == $a1091196985512978262           allow               pkt_mark=1008
+       102 (ip6.src == $a2718358047735721557 || ip6.src == $a13607452020421864338) && ip6.dst == $a3613484745323146040           allow
+       102 ip4.src == 203.203.0.0/16 && ip4.dst == 100.64.0.0/16           allow
+       102 ip4.src == 203.203.0.0/16 && ip4.dst == 203.203.0.0/16           allow
+       102 ip6.src == 2010:100:200::/60 && ip6.dst == 2010:100:200::/60           allow
+       102 ip6.src == 2010:100:200::/60 && ip6.dst == fd98::/64           allow
+       102                                     pkt.mark == 42           allow
+```
+
+And these will be the gateway router configured routes (no policies needed):
+```
+IPv4 Routes
+Route Table <main>:
+           169.254.0.0/17               169.254.0.4 dst-ip rtoe-GR_test12_namespace.scoped_ovn-control-plane
+           203.203.0.0/16               100.88.0.13 dst-ip rtotr-GR_test12_namespace.scoped_ovn-control-plane <-- cluster ingress or egress reply traffic going towards the pod network via transit_router peer IP.
+                0.0.0.0/0                172.18.0.1 dst-ip rtoe-GR_test12_namespace.scoped_ovn-control-plane
+```
+
+For the routes related to EIP, they are moved from gateway router to transit_router and the redirect action is changed to point to the transit node peer ip.
```

Comment:
so this part of nexthop change is definite disruption?
also if the next hop is the transit node peer ip then are we going to have remote routes to then take it up to the remote joinIP ? I think having a section for EIP to show one example route might make things a bit clear